src/Connection.php

```diff
@@ -129,19 +129,16 @@ public function fetchAll(string $sql, array $bind = []): array
         }
     }
 
-    public function fetchRoles(?string $roleLike = null): array
+    public function existsRole(string $role): bool
     {
-        $sql = 'SELECT * FROM INFORMATION_SCHEMA.APPLICABLE_ROLES';
-        $args = [];
-        if ($roleLike !== null) {
-            $sql .= ' WHERE ROLE_NAME = %s';
-            $args[] = $this->quote($roleLike);
-        }
+        $sql = 'SELECT COUNT(*) AS CNT FROM INFORMATION_SCHEMA.ENABLED_ROLES  WHERE ROLE_NAME = %s';
```

Comment:
Query `INFORMATION_SCHEMA.APPLICABLE_ROLES` stale pada na SNFK incidente, ... support to riesi, ale nejak sme sa nepohli zatial, .... preto som hladal ine riesenia, a zistil som, ze v tomto pripade mozeme pouzit aj `INFORMATION_SCHEMA.ENABLED_ROLES`, ktora ide dobre. Klient na to caka, tak to chcem spravit takto.